--- language-org.py	2023-04-04 13:27:16.590000000 +0900
+++ language.py	2023-04-04 13:19:11.970000000 +0900
@@ -124,7 +124,8 @@
     name="python",
     display_name="Python",
     extension="py",
-    submission_lang_pattern=re.compile(".*Python3.*|^Python$"),
+    submission_lang_pattern=re.compile(".*PyPy3.*|^Python$"),
+    # submission_lang_pattern=re.compile(".*Python3.*|^Python$"),
     default_code_generator=python.main,
     default_template_path=get_default_template_path('py'),
     compile_command="python3 -mpy_compile {filename}.py",
